pyvista/plotting/render_window_interactor.py

```diff
@@ -22,6 +22,40 @@
 _CLASSES = {}
 
 
+class Timer:
+    """Timer class.
+
+    Parameters
+    ----------
+    max_steps : int
+        Maximum number of steps for integrating a timer.
+
+    iren : RenderWindowInteractor
+        Rendering win and interactor.
+
+    callback : callable
+        A callable that takes one arguments (step argument is need).
```

Comment:
```suggestion
        A callable that takes one argument. It will be passed `step`, 
        which is the number of times the timer event has occurred.
```